examples/lib/exec-send-recv.go

```diff
@@ -61,7 +61,7 @@ func (obj *MyGAPI) Cli(c *cli.Context, fs engine.Fs) (*gapi.Deploy, error) {
 			Noop: c.GlobalBool("noop"),
 			Sema: c.GlobalInt("sema"),
 			GAPI: &MyGAPI{
-			// TODO: add properties here...
+				// TODO: add properties here...
```

Comment:
You can probably fix these with `make gofmt` from the project root. Otherwise you can `git checkout examples/lib/exec-send-recv.go` with the path of each changed file and amend the commit.

Reply:
Hey Jonathan, thanks a ton for the reply. Sounds a bit embarassing, but running make gofmt tells me "make: *** No rule to make target 'gofmt'.  Stop." and I am not sure how to run the command differently. I've tried gofmt (file name), but I am not sure if that accomplishes the same function.

Reply:
Did you try running it from `$GOPATH/src/github.com/purpleidea/mgmt`? (It won't work from subdirectories)

Reply:
This is what i ran:

```[michael@localhost mgmt]$ cd $GOPATH/src/github.com/ginywiny/mgmt
[michael@localhost mgmt]$ pwd
/home/michael/gopath/src/github.com/ginywiny/mgmt
[michael@localhost mgmt]$ make gofmt
# TODO: remove gofmt once goimports has a -s option
find . -maxdepth 6 -type f -name '*.go' -not -path './old/*' -not -path './tmp/*' -not -path './vendor/*' -exec gofmt -s -w {} \;
find . -maxdepth 6 -type f -name '*.go' -not -path './old/*' -not -path './tmp/*' -not -path './vendor/*' -exec goimports -w {} \;
```

Is this correct, or must it be run directly from purpleidea's directory rather than ginywiny's?

Reply:
As long as you're in the correct branch it should be fine.

Caveat: If `make gofmt` invokes go 1.10's gofmt, it won't fix the issue. It may be easier to checkout each affected file from master. 

Or, make a backup copy of `pgraph/graphsync_test.go` (outside of the project directory,) and reset your branch with `git fetch origin` followed by `git reset --hard origin/master`. Then you can copy your backup back into `pgraph/` then `git add -p` to add the changes and `git commit` followed by `git push -f`. Don't do this without making sure you have backed up the file outside `mgmt/`, or you will lose your work.